internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go

```diff
@@ -68,7 +71,7 @@ func (l *Manager) Wakeup(ctx context.Context, appID api.AppID) error {
 	item.nextRequestAfter = now.Add(Interval)
 
 	// Send the notification
-	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil {
+	if _, err := l.api.WakeupApp(appID).Send(ctx); err != nil && err.Error() != wakeupErrorToBeSkipped {
```

Comment:
So you want to just ignore this specific error and behave as though wakeup succeeded? I don't like that tbh. Maybe just use lower log level then error instead? Also add some comment why we're treating this case differently.

Reply:
c971ed3496ef7dee7a7599cb1778552e53d94ee4